_posts/2015-09-16-introduction-of-recommender-algorithms-for-e-commerce.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "Introduction of Recommender Algorithms for E-Commerce"
+date: "2015-09-16"
+icon: Light-bulb
+categories: tech-stories
+authors: ["Pujit"]
+---
+
+##Introduction
+
+A recommender system in e-commerce domain can be seen as equivalent of sales personal. Like a good sales personal in a store helps increasing sales and building loyalty customer base, recommender system handles sales three ways: browser into buyers, cross-sell and loyalty.
+
+* **Browsers into buyers:** Visitors to a Web site often look over the site without ever purchasing anything. Recommender systems can help customers find products they wish to purchase.
+
+* **Cross-sell:** Recommender systems improve cross-sell by suggesting additional products for the customer to purchase. If the recommendations are good, the average order size should increase. For instance, a site might recommend additional products in the checkout process, based on those products already in the shopping cart.
+
+* **Loyalty:** In a world where a site’s competitors are only a click or two away, gaining customer loyalty is an essential business strategy. [4,5] Recommender systems improve loyalty by creating a value-added relationship between the site and the customer. Sites invest in learning about their users, use recommender systems to operationalize that learning, and present custom interfaces that match customer needs. Customers repay these sites by returning to the ones that best match their needs. [3]
```

Comment:
Afaik you don't need the last comma in the second to last sentence of that block, because there's the "and": "(...) use recommender systems to operationalize that learning and present custom interfaces (...)" It might be even wrong to have that comma.